### DISCUSSION TITLE
Why does repository.apache.org use an old version of Nexus Repository Manager?

### DISCUSSION THREAD

**Discussion by linghengqian** [apache/infrastructure-ideas#6] [Q&A] [answered]
- Of course, I would say this question is just my personal curiosity. 
- https://repository.apache.org/ is using Nexus Repository Manager 2.14.20-02 . And https://help.sonatype.com/en/repository-manager-2.html mentions,

> Sonatype will officially sunset its Nexus Repository 2 product on June 30, 2025. If you are using Nexus Repository 2, you should [migrate to Sonatype Nexus Repository 3](https://help.sonatype.com/en/upgrading-from-nexus-repository-manager-2.html) as soon as possible.
> For full details about and frequently asked questions, see the [Sonatype Nexus Repository 2 Sunsetting Information help page](https://help.sonatype.com/en/sonatype-nexus-repository-2-sunsetting-information.html).

- The interface of Nexus Repository Manager 2.14.20-02 looks a bit old.
- ![image](https://github.com/user-attachments/assets/e43d3ec9-2923-4710-9529-e2364113407e)
- I started a Sonatype Nexus Repository 3.71.0 instance locally through the bash command `docker run -d -p 8081:8081 --name nexus sonatype/nexus3:3.71.0` to test it. It seems that its interface is more intuitive for users.
- ![image](https://github.com/user-attachments/assets/e27b3b95-3d0d-4641-a9bf-f3bf430b1742)
- Why does https://repository.apache.org/ use an old version of Nexus Repository Manager?

**Comment by brianf** [comment, marked as answer]
The staging workflows for Nexus 1 and 2 were heavily inspired by how the Apache Maven project did manual staging in the early days since we are maven devs. That model propagated through Apache and encoded in the staging support for Nexus. This model wasn't compatible with how many enterprises wanted to do promotion and thus Nexus 3 has a more flexible model but it requires external workflow tooling to automate it. 

We are working on replacements to the promotion model that would allow a Nexus 3 upgrade before NX2 is sunset, and of course, we (Sonatype) aren't just going to drop the RAO instance on its face during the sunset.

**Reply by linghengqian** [reply]
Thanks for the clarification, I will close the current discussion.
